### PR TITLE
ContactsController, Actions.

### PR DESCRIPTION
**Pull request by tajwal**

### CODE REVIEW

**Comment by tajwal** [comment]
GetAllContacts, Postman response
GetAll

**Comment by hassanhabib** [comment]
@viralpandya take a look at this PR please.

**Comment by viralpandya** [comment]
> @viralpandya take a look at this PR please.

This PR is good. Good work and keep it up!

Let us add post screenshot as well.

**Comment by tajwal** [comment]
ContactsController, GetContactByIdAsync
GetById

**Comment by tajwal** [comment]
@viralpandya here is screenshop for post.

ContactsController, PostContactAsync.

PostAsync

**Comment by tajwal** [comment]
ContactsController, PutContactAsync
Put

File: OtripleS.Web.Api/Controllers/ContactsController.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ContactsController : RESTFulController
+    {
+        private readonly IContactService contactService;
+
+        public ContactsController(IContactService contactService) => this.contactService = contactService;
+
+        [HttpPost]
+        public async ValueTask<ActionResult<Contact>> PostContactAsync(Contact contact)
```
Comment (tajwal) [inline OtripleS.Web.Api/Controllers/ContactsController.cs:28]: @viralpandya  can you please check, its throwing which doesn't belongs to this controller and service. 

error

**Pull request merged**